recipes/pdsspect/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pdsspect" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "c5207e99bb3131579b18290430390f60373758d98e59e3781c6e30b01ead5359" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like there is a cli. Please add it here. See:

https://conda.io/docs/building/meta-yaml.html#python-entry-points